### PR TITLE
Module 6 Block 1 :: Set up a "Hello world" project with two apps

### PR DESCRIPTION
**Pull request by aceDavon**
## In this project, I have implented the following tasks: :writing_hand: 

- [ ] Create a new Rails API app.
- [ ] Initialize project with Git.
- [ ] Make sure that project has a Postgres database set up.
- [ ] Create a table for storing messages and create 5 different greetings.
- [ ] Create an API endpoint that selects a random greeting from table.

__Front End Link: https://github.com/aceDavon/react-front__

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Davon, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has incorrect gitflow/ GitHub flow. Kindly note that the merge request for this activity should flow **from feature into development**. Kindly use the correct gitflow.

Kindly also ensure to remove the template logo, in your ReadMe. Optionally, you can replace it with your own logo if you have one.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/controllers/roots_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class RootsController < ApplicationController
+  def index
+    @greeting = Root.all.sample
+
+    render json: { greeting: @greeting, status: 200 }
```
Comment (Davidosky007) [inline app/controllers/roots_controller.rb:5]: * Kindly fix this issue I have as the front end doesn't render greetings but the backend API does 
image
image

**Review by Davidosky007 — COMMENTED** [review]
Hi @aceDavon 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Well written PR description 👍 
* Good looking readme file 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @aceDavon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**